common/common-gpu.h

```diff
@@ -30,7 +30,7 @@ POSSIBILITY OF SUCH DAMAGE.
  */
 
 #pragma once
-#include "daemon_config.h"
+#define CONFIG_VALUE_MAX 256
```

Comment:
Why the duplicated definition?

Reply:
To split out the dependency between `common` and `daemon` code.

But yeah, it's a bit confusing now, I'll clean it up.

Reply:
Fixed in d9b9141